### ISSUE TITLE
Can't set dark mode

### ISSUE THREAD

**Issue by videodj** [tgcowell/waves#47]
I followed the instructions and have everything running, however when I select dark mode nothing happens am I missing something?

**Comment by pedropinn** [comment]
Below where you set the theme (dropdown list) there are 3 checkboxes, automatic, light and dark. You should select dark.

**Comment by LegendaryFire** [comment]
@videodj @pedropinn I've also had this issue when setting an automation on startup to set the default theme. Although I select dark, it still loads the light theme instead. I'll take a look in the source code sometime in the next few weeks and see if I can find out what's going on with it. Still waiting on my PR for the color fixes to be merged too.

**Comment by qJake** [comment]
I found this thread because I can't get the light/dark to work either - it is supposed to be automatic on mobile, but it's defaulting to light.

Honestly, it would be much easier if there was just a "waves-light" and "waves-dark" theme in the dropdown separately so that we can just pick one. (I'm using this as a dashboard theme only, not a system-wide theme, so I don't even have the Light/Dark/Automatic toggle.)

**Comment by tgcowell** [comment]
Hi All,

Any chance you could give me any additional info? I have cannot replicate the issue, it changes without any concerns.

Not sure how to even start troubleshooting

**Comment by qJake** [comment]
So, I think this is less of a "bug" and more of a request to add additional functionality or otherwise enhance the theme.

Currently, if you set the theme system-wide, you do get the Light/Dark/Auto toggle. That works fine.

But if you want to only use waves on a single dashboard, or a single tab, you can't do that - it defaults to light only. You *can't* use the dark theme, as far as I can tell (unless I'm missing something obvious?).

![image](https://github.com/tgcowell/waves/assets/652006/367674c3-75f4-4999-abe0-82bd62f8c23a)

My request is to have separate waves-light and waves-dark themes so I can pick one specifically. I'm not sure if that's the same thing that @videodj is reporting, if not, I will submit a separate issue!

**Comment by freeman59** [comment]
Hello there, I have a different problem. If I put the wave dark in my profile, it works but clear not. I mean it works but I don't have the background for this setting. I tried also to upload a background image when using Wave in a dashboard but it doens't show up either. I've switched to defaut theme on my profile then back to dashboard and put the wave theme, same problem. If I put a specific image on background with a defaut theme, it works so Idk why it's not working when selecting the wave theme

do you have an idea ?

regards
